"flag redefined: cpuprofile" with sha1cd v0.3.1, avo v0.6.0

Hi, you are using `github.com/pjbgf/sha1cd` currently in verion 0.3.0. If upgraded to version 0.3.1 it also [updates](https://github.com/pjbgf/sha1cd/commit/098274f2b97e751922814710e15dde109dee3a01#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R5) `github.com/mmcloughlin/avo` to v0.6.0, which defines the flag "cpuprofile" [here](https://github.com/mmcloughlin/avo/blob/master/build/cli.go#L71)  also defined here: https://github.com/sourcegraph/zoekt/blob/27f6d0b06f93595281406b7b8c5d6e20a0df6163/cmd/zoekt-git-index/main.go#L49

Its probably a good idea to create some own FlagSet ([NewFlagSet](https://pkg.go.dev/flag#NewFlagSet)) or do some workaround to keep zoekt updatable in the future.

Maybe it would even be possible to remove the old workaround "cpu_profile" by doing it right this time and consolidate the cpuprofile argument. https://github.com/sourcegraph/zoekt/blob/27f6d0b06f93595281406b7b8c5d6e20a0df6163/cmd/zoekt-index/main.go#L62


**Steps to reproduce** 

`go get -u ./... && go run ./cmd/zoekt-git-index/`

**Output:**

```
/tmp/go-build2071650996/b001/exe/zoekt-git-index flag redefined: cpuprofile
panic: /tmp/go-build2071650996/b001/exe/zoekt-git-index flag redefined: cpuprofile

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0000f6150, {0x13140e0, 0xc0002f70c0}, {0x10b0cbb, 0xa}, {0x11935ee, 0x1b})
        /opt/go/src/flag/flag.go:1028 +0x37d
flag.(*FlagSet).StringVar(...)
        /opt/go/src/flag/flag.go:879
flag.(*FlagSet).String(0xc0000f6150, {0x10b0cbb, 0xa}, {0x0, 0x0}, {0x11935ee, 0x1b})
        /opt/go/src/flag/flag.go:892 +0xa5
flag.String(...)
        /opt/go/src/flag/flag.go:899
main.run()
        /home/user/github/zoekt/cmd/zoekt-git-index/main.go:49 +0x25a
main.main()
        /home/user/github/zoekt/cmd/zoekt-git-index/main.go:146 +0x13
```

--- a/cmd/zoekt-git-index/main.go
+++ b/cmd/zoekt-git-index/main.go
@@ -46,7 +46,7 @@ func run() int {
 	deltaShardNumberFallbackThreshold := flag.Uint64("delta_threshold", 0, "upper limit on the number of preexisting shards that can exist before attempting a delta build (0 to disable fallback behavior)")
 	languageMap := flag.String("language_map", "", "a mapping between a language and its ctags processor (a:0,b:3).")
 
-	cpuProfile := flag.String("cpuprofile", "", "write cpu profile to `file`")
+	cpuProfile := flag.String("cpu_profile", "", "write cpu profile to `file`")
 
 	flag.Parse()
 

--- a/cmd/zoekt-test/main.go
+++ b/cmd/zoekt-test/main.go
@@ -170,8 +170,8 @@ func compare(dir, patfile string, caseSensitive bool) error {
 }
 
 var (
-	memprofile = flag.String("memprofile", "", "write memory profile to `file`")
-	cpuprofile = flag.String("cpuprofile", "", "write memory profile to `file`")
+	memprofile = flag.String("mem_profile", "", "write memory profile to `file`")
+	cpuprofile = flag.String("cpu_profile", "", "write cpu profile to `file`")
 )
 
 func testLoadIndexDir(indexDir string) {

